clients/python/tests/conftest.py

```diff
@@ -133,3 +134,19 @@ def event_loop():
 @cleanup
 def client() -> ModelRegistry:
     return ModelRegistry(REGISTRY_HOST, REGISTRY_PORT, author="author", is_secure=False)
+
+@pytest.fixture(scope="module")
+def setup_env_user_token():
+    token_file = tempfile.NamedTemporaryFile(delete=False)
+    token_file.write(b"Token")
+    token_file.close()
```

Comment:
```suggestion
    with tempfile.NamedTemporaryFile(delete=False) as token_file:
        token_file.write(b"Token")
```

possibly more pythonic